### ISSUE TITLE
Improve the listbox label

### ISSUE THREAD

**Issue by afercia** [swissspidy/preferred-languages#13]
In `profile.php` the text "Language" on the left column shouldn't be a `<label>` element. In the General Settings screen it is not a label (correctly).

Listboxes do need a _visible_ label though, as specified in the WAI-ARIA Authoring Practices:
https://www.w3.org/TR/wai-aria-practices/#Listbox
> If the listbox is not part of another widget, then it has a visible label referenced by aria-labelledby on the element with role listbox.

Examples:
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox.html

This basically means the current `aria-label="Order locales in order of preference"` should be removed in favor of a visible element targeted by `aria-labelledby`. Also, the label value (whether it's provided via aria-label, aria-labelledby, or an actual label element for controls that require it) represents the accessible name of a control. Should be used to give the control a name. Instead, instructions/hints, if needed, can be provided targeting an existing element with `aria-describedby`.

**Comment by swissspidy** [comment]
Great suggestion, thanks!

**Comment by afercia** [comment]
Sorry I was unclear :) `aria-labelledby` must be used to target a _visible_ element, and that element shouldn't be a `<label>`. In the examples linked above, they're just `<span>` elements.

Ideally, the text in the left column should be used for this purpose; unfortunately the Settings API doesn't allow to add an `id` there, unless passing some HTML to the `title`. This would be easier in the user profile page, since there's more control over the HTML output.

<img width="774" alt="screen shot 2017-05-21 at 13 28 51" src="https://cloud.githubusercontent.com/assets/1682452/26283489/0e1fb8f0-3e2a-11e7-9a22-6dffc6c13f96.png">

Ideally, the highlighted text ^^ should be used as target for `aria-labelledby`.

**Comment by swissspidy** [comment]
Thanks again for your help here @afercia. I fixed this in 2522703def0d205e51585bf655c1368fdfa7a4ce by using the text you highlighted in the screenshot for the label.
